docs_src/microservices/device/supported/device-usb-camera/supplementary-info/discovery.md

```diff
@@ -45,7 +45,7 @@ Dynamic discovery is enabled by default to make setup easier. It can be disabled
     export DEVICE_DISCOVERY_INTERVAL=1h
     ```
 
-To manually trigger a Dynamic Discovery, use this [device service API](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/device-sdk/2.2.0#/default/post_discovery).  
+To manually trigger a Dynamic Discovery, use this [device service API](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/device-sdk/3.0.0#/default/post_discovery).  
```

Comment:
This should link to docs page for SDK's swagger here: https://docs.edgexfoundry.org/3.0/api/devices/Ch-APIDeviceSDK/